automod/triggers.go

```diff
@@ -47,6 +48,12 @@ func (r BaseRegexTrigger) UserSettings() []*SettingDef {
 			Min:  1,
 			Max:  250,
 		},
+		{
+			Name:    "Also match Zalgo and confusable characters",
```

Comment:
Make it simple for a average user? What does Zalgo and confusable characters here mean?

Reply:
Yeah I just stole this text from Luca